neurots/schemas/parameters.json

```diff
@@ -117,22 +117,42 @@
                                             },
                                             "additionalProperties": false,
                                             "required": ["primary_orientation"]
-                                         },
-                                          {
+                                        },
+                                        {
                                             "type": "object",
                                             "properties": {
                                                 "direction": {
-                                                    "maxItems": 2,
-                                                    "minItems": 2,
-                                                    "items": {
-                                                        "description": "The orientation 3d angle (mean, std)"
-                                                    },
-                                                    "type": "array"
+                                                    "type": "object",
+                                                    "properties": {
+                                                        "mean": {
+                                                            "oneOf": [
+                                                                {
+                                                                    "type": "number",
+                                                                    "minimum": 0,
+                                                                    "maximum": 3.1416
+                                                                },
+                                                                {
+                                                                    "type": "array"
```

Comment:
When it is an array, what is it's structure? Is it an array of numbers between 0 and pi? If yes we should also constrain the items of this array.

Reply:
yes, list of such numbers, when there are more than one apical. How to do that?

Reply:
@adrien-berchet, like that?

Reply:
Yep, thanks :)